### PR TITLE
updated the logging in the notify controller

### PR DESCRIPTION
**Pull request by bkhan2**
# Motivation and Context
updating logging when notify notification doesn't get sent to be clearer

# What has changed
added URL and Status Code logging.

### CODE REVIEW

File: frontstage/controllers/notify_controller.py [hunk 1]
```diff
@@ -41,26 +41,27 @@ def _send_message(self, email, template_id, personalisation=None, reference=None
...
+        auth = app.config['SECURITY_USER_NAME'], app.config['SECURITY_USER_PASSWORD']
+        response = requests.post(url, json=notification, auth=auth,
+                                 timeout=int(app.config['REQUESTS_POST_TIMEOUT']))
+        status_code = response.status_code
 
+        if status_code == 201:
             logger.info('Notification id sent via Notify-Gateway to GOV.UK Notify.', id=response.json()["id"])
-
-        except Exception as e:
+        else:
```
Comment (jterryons) [inline frontstage/controllers/notify_controller.py:60]: Silimar comment to the other pr. we shouldbe be raising an error on codes 200 -399.

Comment (mfcrocker) [reply frontstage/controllers/notify_controller.py:60]: As other PR, disagree with this

File: frontstage/controllers/notify_controller.py [hunk 2]
```diff
@@ -41,26 +41,27 @@ def _send_message(self, email, template_id, personalisation=None, reference=None
...
+                                 timeout=int(app.config['REQUESTS_POST_TIMEOUT']))
+        status_code = response.status_code
 
+        if 201 <= status_code <= 399:
             logger.info('Notification id sent via Notify-Gateway to GOV.UK Notify.', id=response.json()["id"])
-
-        except Exception as e:
+        else:
             ref = reference if reference else 'reference_unknown'
             raise exceptions.RasNotifyError("There was a problem sending a notification to Notify-Gateway "
```
Comment (mfcrocker) [inline frontstage/controllers/notify_controller.py:62]: You don't need to pass these variables in as fstrings, `RasNotifyError` takes kwargs
```suggestion
            raise RasNotifyError("There was a problem sending a notification to Notify-Gateway.", url=url, status_code=status_code, message=response.text, reference=ref)
```

**Comment by jterryons** [comment]
the code can only be tested in a suite of integration tests. None of the code is isolated for testing making it very difficult to trust any of the changes for so little gain. its not worth the effort right now, we have bigger issues to deal with.

**Pull request closed without merging**